docs/manual/upgrades.md

```diff
@@ -148,6 +148,29 @@ We discussed in [Process restarted during staging Phase](#process-restarted-duri
 For version incompatible upgrades the operator has no way to differentiate between processes not reporting to the cluster and processes running in the incompatible version, both will be missing from the processes list.
 The only difference is that the incompatible processes might be reported in the `incompatible_connections` field.
 
+#### Processes are not able to join the FDB cluster
+
+We've seen cases where a subset of `fdbserver` were not able to join the FDB cluster.
+This can happen due to a race condition where a new set of coordinators is chosen.
```

Comment:
If you want to provide more details here you can say: "the server process (that is not able to join) has a stale connection string, and a majority of coordinators in its connection string have restarted".